src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/MultiSelectList.cs

```diff
@@ -10,6 +10,11 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Represents a list that lets users select one item.
```

Comment:
a) This _is_ a list.
b) This particular list lets users select _multiple_ items.